src/__tests__/compilesb3.test.ts

```diff
@@ -3,16 +3,22 @@ import { Project } from "..";
 import * as fs from "fs";
 import * as path from "path";
 
-test("sb3 -> scratch-js", async () => {
-  const file = fs.readFileSync(path.join(__dirname, "test.sb3"));
-  const project = await Project.fromSb3(file);
+async function loadProject(filename) {
+  const file = fs.readFileSync(path.join(__dirname, filename));
+  return await Project.fromSb3(file);
```

Comment:
`return await` is [apparently unnecessary/an antipattern](https://eslint.org/docs/rules/no-return-await).